libnmstate/varlink/io.nmstate.varlink

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# This file is part of nmstate
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+# Interface definition
+interface io.nmstate
+
+# Method definition
+method Show(arguments: [string]object) -> (
+    state: ?object,
+    log: []object
+)
+
+method Apply(arguments: [string]object) -> (
+    result: ?object,
```

Comment:
The result should be `enum pass|fail` instead of external object.

Reply:
If the exception is handled by errors, maybe we don't need this `result`. Just log is enough.

Reply:
@cathay4t 
Exceptions are handled by varlink errors so I'll remove the result parameter